src/main/java/edu/ucla/library/iiif/fester/Constants.java

```diff
@@ -112,6 +112,17 @@
      */
     public static final String IIIF_HOST = "iiif-host";
 
+    /**
+     * Used as a message header when the sender wants the manifest returned from S3 without resource URLs re-written.
+     */
+    public static final String NO_REWRITE_URLS = "no-rewrite-urls";
+
+    /**
+     * A unique random placeholder URL that prefixes all IIIF Presentation API resource URLs in all manifests at rest in
+     * S3. It gets replaced with Constants.URL on each GET request.
+     */
+    public static final String URL_PLACEHOLDER = "http://b1dbe4a0-443c-479f-bf0a-25c352df0d8f.example.org";
```

Comment:
Maybe use a UUID + a UCLA library domain (e.g. http://b1dbe4a0-443c-479f-bf0a-25c352df0d8f.iiif.library.ucla.edu)? I know the example.org domain is for examples, but I'm just a little hesitant to use it for a placeholder since we don't own it.